docs/rest/src/files/CheckFileInfo.rst

```diff
@@ -212,6 +212,8 @@ WOPI implementation meets the requirements for a particular property.
 
         ..  versionadded:: 2015.08.03
 
+    TemporarilyNotWritable
+        A **Boolean** value that indicates that if host is temporarily unable to process writes on a file
```

Comment:
This should be in the miscellaneous properties section. It doesn't represent a host capability.